FreeRTOS/FreeRTOS
# Add Project for running integration tests

Description
-----------
This PR is part of a bigger effort to enable testing of +TCP library in the FreeRTOS/FreeRTOS repository. The project introduced in this PR enables the user to run the integration tests present [here](https://github.com/FreeRTOS/FreeRTOS/tree/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/test).

Test Steps
-----------
Just download the repo with --recurse-submodules flag and open the Visual Studio project (.sln file) present in `FreeRTOS-Plus/Test/FreeRTOS_Plus_TCP_Integration_Tests`. Make sure that the computer is connected to a network via ethernet cable and choose the required network interface by modifying this line `#define configNETWORK_INTERFACE_TO_USE` in `FreeRTOSConfig.h`. 

Once these changes are made, just build and run the project. It should run 4 test of which all should pass. 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
